liaisons.md

```diff
@@ -1,3 +1,65 @@
-## Liaisons to the W3C WoT Community Group
+# Liaisons to the W3C WoT Community Group
 
-To be decided
+The liaisons of the W3C WoT Community Group (CG) are not explicitly defined in the CG Charter.
+However, they are documented in this document.
+This document describes each liaison and defines the scope of responsibility.
+
+Current liaison:
+
+- [Web of Things Working Group](#web-of-things-working-group)
+
+## Web of Things Working Group
+
+In this section, WG refers to the Web of Things Working Group.
+CG refers to the Web of Things Community Group.
+
+> [!NOTE]
+> This liaison is a draft until a resolution is reached in both the WG and CG.
+> Resolutions will be documented via email.
+> The CG Charter will need to be updated to satisfy this liaison.
+> Agreement on this document implies agreement on the corresponding CG Charter updates.
+> Until the revised CG Charter becomes active, this liaison remains in draft status.
+
+This liaison defines the role of the CG with respect to the work of the WG.
+The CG has four primary roles: facilitating a low entry barrier, conducting outreach, organizing plugfests, and developing bindings.
+
+### Facilitating a Low Entry Barrier
+
+Because WG participation requires W3C membership, it is important to foster broader engagement and idea exchange on WoT-related topics.
```

Comment:
I don't think it's necessary to refer to Membership here. The CG and the WG should be doing different things, and someone should get involved with either group depending on one's interests.

Reply:
Patent policy yes afaik. Charter can also happen, never seen it yet. Of course WG Charter and WG-level patent policy

Reply:
Working Groups should not accept contributions from non-participants unless the contributors make licensing commitments. But, in addition, if a CG participant contributes something to a CG, they are making the same RF licensing commitment as they would be making in the WG, and if the work is taken up by the WG, their commitment follows it.